test/test_core.py

```diff
@@ -95,10 +97,12 @@ def test_from_import_non_existing_julia_name(self):
             assert not spamspamspam
 
     def test_julia_module_bang(self):
-        from julia import Base
-        xs = [1, 2, 3]
-        ys = Base.scale_b(xs[:], 2)
-        assert all(x * 2 == y for x, y in zip(xs, ys))
+        from julia.Base import Channel, put_b, take_b
+        chan = Channel(1)
+        sent = 123
+        put_b(chan, sent)
+        received = take_b(chan)
+        assert sent == received
```

Comment:
I go ahead and I do https://github.com/JuliaPy/pyjulia/pull/183#discussion_r211484573 (to make sure that it works on CIs). If you prefer `fill_b` or something else let me know.

If CIs and you are happy with it then I think it's ready for merge.